index.js

```diff
@@ -49,6 +49,13 @@ class LedgerBridgeKeyring extends EventEmitter {
     this.accounts = opts.accounts || []
     this.accountIndexes = opts.accountIndexes || {}
     this.implementFullBIP44 = opts.implementFullBIP44 || false
+
+    if (this._isBIP44()) {
+      // Remove accounts that don't have corresponding account indexes
+      this.accounts = this.accounts
+        .filter(account => this.accountIndexes[account])
```

Comment:
Should we be checksumming these in the lookup?

Reply:
Yes, we should great catch!

No idea if we persist them as checksummed addresses 🤔 but best be safe for now

Reply:
While I'm at it, I'll add an explicit `Boolean(..)` so it's clear it's being interpreted as a bool.